README.md

```diff
@@ -154,11 +167,10 @@ It has an unfortunate overlap with React's [TSX/JSX](https://www.typescriptlang.
 
 ### Does it do type-checking?
 
-No, [esbuild does not support type checking](https://esbuild.github.io/faq/#:~:text=TypeScript%20type%20checking%20(just%20run%20tsc%20separately)).
+No, [esbuild does not support type checking](<https://esbuild.github.io/faq/#:~:text=TypeScript%20type%20checking%20(just%20run%20tsc%20separately)>).
```

Comment:
Why?

Reply:
vscode auto format caused of that, I'll revert unnecessary changes asap...